packages/grafana-ui/src/components/Combobox/MultiCombobox.tsx

```diff
@@ -26,15 +27,19 @@ import { getMultiComboboxStyles } from './getMultiComboboxStyles';
 import { useComboboxFloat } from './useComboboxFloat';
 import { useMeasureMulti } from './useMeasureMulti';
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const __ALL: any = 'all';
```

Comment:
```suggestion
const ALL_OPTION_VALUE: string = '__all__';
```

To prevent a certain risk to interfere with any manually set values. Not sure what you can do with the typing though.